pkgs/development/tools/azcopy/default.nix

```diff
@@ -1,14 +1,14 @@
-{ stdenv, lib, fetchFromGitHub, buildGoModule }:
+{ stdenv, lib, fetchFromGitHub, buildGoModule, azure-storage-azcopy, testers }:
```

Comment:
This looks like it should be removed.

Reply:
Good catch, fixed and squashed in https://github.com/NixOS/nixpkgs/pull/250476/commits/19154f8b9effbad10db55b44cb773073352a7251